optimum/onnxruntime/trainer.py

```diff
@@ -907,7 +907,7 @@ def evaluate(
         """
         # memory metrics - must set up as early as possible
         # TODO: We need to enable evaluation using ORT backend.
-        self.model = unwrap_model(self.model)
+        self.model = self.model._original_model
```

Comment:
don't we need a check module is loss is enabled? otherwise, how thye have ._original model????

Reply:
@AdamLouly Can we add a checker here for the case without the `ModuleWithLoss` wrapper?

Reply:
Good catch, we have to check here.